plugins/connection/pct_remote.py

```diff
@@ -0,0 +1,396 @@
+        ini:
+            - {key: use_rsa_sha2_algorithms, section: paramiko_connection}
+        env:
+            - {name: ANSIBLE_PARAMIKO_USE_RSA_SHA2_ALGORITHMS}
+        default: True
+        type: boolean
+      host_key_auto_add:
+        description: 'Automatically add host keys'
+        env: [{name: ANSIBLE_PARAMIKO_HOST_KEY_AUTO_ADD}]
+        ini:
+          - {key: host_key_auto_add, section: paramiko_connection}
+        type: boolean
+      look_for_keys:
+        default: True
+        description: 'False to disable searching for private key files in ~/.ssh/'
+        env: [{name: ANSIBLE_PARAMIKO_LOOK_FOR_KEYS}]
+        ini:
+        - {key: look_for_keys, section: paramiko_connection}
+        type: boolean
+      proxy_command:
+        default: ''
+        description:
+            - Proxy information for running the connection via a jumphost
+            - Also this plugin will scan 'ssh_args', 'ssh_extra_args' and 'ssh_common_args' from the 'ssh' plugin settings for proxy information if set.
+        type: string
+        env: [{name: ANSIBLE_PARAMIKO_PROXY_COMMAND}]
+        ini:
+          - {key: proxy_command, section: paramiko_connection}
+        vars:
+          - name: ansible_paramiko_proxy_command
+      ssh_args:
+        description: Only used in parsing ProxyCommand for use in this plugin.
+        default: ''
+        type: string
+        ini:
+            - section: 'ssh_connection'
+              key: 'ssh_args'
+        env:
+            - name: ANSIBLE_SSH_ARGS
+        vars:
+            - name: ansible_ssh_args
+        deprecated:
+            why: In favor of the "proxy_command" option.
+            version: "0.1.0"
+            alternatives: proxy_command
+      ssh_common_args:
+        description: Only used in parsing ProxyCommand for use in this plugin.
+        type: string
+        ini:
+            - section: 'ssh_connection'
+              key: 'ssh_common_args'
+        env:
+            - name: ANSIBLE_SSH_COMMON_ARGS
+        vars:
+            - name: ansible_ssh_common_args
+        cli:
+            - name: ssh_common_args
+        default: ''
+        deprecated:
+            why: In favor of the "proxy_command" option.
+            version: "0.1.0"
+            alternatives: proxy_command
+      ssh_extra_args:
+        description: Only used in parsing ProxyCommand for use in this plugin.
+        type: string
+        vars:
+            - name: ansible_ssh_extra_args
+        env:
+          - name: ANSIBLE_SSH_EXTRA_ARGS
+        ini:
+          - key: ssh_extra_args
+            section: ssh_connection
+        cli:
+          - name: ssh_extra_args
+        default: ''
+        deprecated:
+            why: In favor of the "proxy_command" option.
+            version: "0.1.0"
+            alternatives: proxy_command
+      pty:
+        default: True
+        description: 'SUDO usually requires a PTY, True to give a PTY and False to not give a PTY.'
```

Comment:
```suggestion
        description: 'C(sudo) usually requires a PTY, V(True) to give a PTY and V(False) to not give a PTY.'
```
It might be more "neutral" to refer to it as the _become mechanism_ rather than explicitly naming it `sudo` - some people use different privilege escalation tools.
And to the best of my knowledge, `sudo` only requires the PTY when it needs to prompt for a password. When using `NOPASSWD` that is not required. Please note that whichever command is being executed by sudo might require a PTY itself.

Reply:
I also copied this from the `ssh_paramiko` docs. I cloud change it, but then it would be inconsistent with `ssh_paramiko`. I think the best case would be if we could inherent the docs from the base class, but that is afaik impossible. The only way I found would be to use `extends_documentation_fragment` but to really share them we would need to edit `ssh_paramiko` as well.